src/interactions/morse.jl

```diff
@@ -0,0 +1,32 @@
+export Morse
+
+"""
+    Morse
+
+Morse potential for bonds between two atoms
+
+"""
+
+struct Morse{T, A, R} <: SpecificInteraction
```

Comment:
Maybe call it `MorseBond` to match `HarmonicBond`?

Reply:
Done.